mkdocs.yml

```diff
@@ -7,7 +7,14 @@ nav:
     - 'Installation': 'common/installation.md'
     - 'YAML Input Specification': 'common/input.md'
     - 'MMS Driver': 'common/mms.md'
-    - 'Running RDycore Examples on Supported DOE Machines': 'user/example-cases/example-cases.md'
+    - 'Examples Cases':
```

Comment:
Let's use 'Example Cases' instead of 'Examples Cases'

Reply:
updated